stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1155,6 +1156,69 @@ class Stripe internal constructor(
         )
     }
 
+    /**
+     * Creates a single-use token that represents the details for a person. Use this when creating or
+     * updating persons associated with a Connect account.
+     * See [the documentation](https://stripe.com/docs/connect/account-tokens) to learn more.
+     *
+     * Person tokens may be created only in live mode, with your application’s publishable key.
```

Comment:
Person tokens can be created in live or test mode using the publishable key. And the mobile SDKs don't use secret keys, so maybe we can remove these two comments?